src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -1,18 +1,116 @@
 package wolox.training.controllers;
 
-import org.springframework.stereotype.Controller;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import wolox.training.exceptions.BookIdMismatchException;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
 
-@Controller
+/**
+ * BookController.java Controller for Books
+ *
+ * @author M. G.
+ */
+
+@RestController
+@RequestMapping("/api/books")
 public class BookController {
 
+    @Autowired
+    private BookRepository bookRepository;
+
+    /**
+     * Greet a person.
+     *
+     * @param name the name of the person
+     * @return the name of the view rendered
+     */
+
     @GetMapping("/greeting")
     public String greeting(
         @RequestParam(name = "name", required = false, defaultValue = "World") String name,
         Model model) {
         model.addAttribute("name", name);
         return "greeting";
     }
+
+    /**
+     * Find all books.
```

Comment:
You can use here something like this:
```
    /**
     * Find all {@link Book}s.
```

So there will be a link to the Book class :) 
Same at the rest of the Javadocs